site-src/api-types/httproute.md

```diff
@@ -43,6 +43,64 @@ spec:
 Note that the target Gateway needs to allow HTTPRoutes from the route's
 namespace to be attached for the attachment to be successful.
 
+You can also attach routes to specific sections of the parent resource.
```

Comment:
Not a blocker or required by any means, but it _might_ be helpful to have some section headings somewhere in here to make it a bit easier to link to a specific section. Not quite necessary yet, but if we end up adding more content here in response to the rest of this review, it might become more helpful.

Reply:
I could add a couple of subsections for sectionName and port attachment, but I agree that it might not be very helpful or needed unless the section gets longer.